src/lwc/geBatchGiftEntryHeader/geBatchGiftEntryHeader.js

```diff
@@ -62,4 +62,17 @@ export default class GeBatchGiftEntryHeader extends LightningElement {
     get qaLocatorEditBatchInfo() {
         return `button ${this.LABELS.geEditBatchInfo}`;
     }
+
+    get processBatchButtonName() {
+        let buttonName = this.LABELS.bgeProcessBatch;
+        if (this.batchTotals.authorizedPaymentsCount) {
+            buttonName = this.LABELS.bgeProcessBatchAndPayments;
+        }
+        
+        return buttonName;
+    }
+
+    get processBatchButtonAltName() {
+        return `${this.processBatchButtonName} button`; 
```

Comment:
Also, @melissabarber - one final question.. do you think we need to localize the word "button" on the alt-text for the "Process Batch" button. Right now, the ask is to have the button alt-text be either:

Process Batch button 

OR

Process Batch and Payments button 

In both scenarios, everything leading up to the word "button" would be localized and I'm wondering if it makes sense to ensure that _everything_ is localized here. Thanks in advance for your feedback!